### PR TITLE
Adds to the loadout four types of fluff oddities.

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request
image
Four types of oddities (That are quite literally worthless to level up. One stat. Are not randomized rounstart and can't really have perks either) that are mostly fluff and great for characters tell tales and RP with it.

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

**Comment by cdb-is-not-good** [comment]
Not really sure why they'd be oddities, this just seems like something that psions will use to powergamer for round start weapons/tools.

File: modular_sojourn/general/oddity.dm [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+/obj/item/oddity/roundstart/book_2
+	name = "old book"
+	desc = "A book depicting a specific type of story , by using narratives and images. Theres alot of general informations that teach the reader about old and ancient traditions and tales."
+	icon_state = "rounstart_oddity_2"
+
+	oddity_stats = list(
+		STAT_MEC = 1,
+		STAT_BIO = 1,
+		STAT_COG = 1,
```
Comment (Trilbyspaceclone) [inline modular_sojourn/general/oddity.dm:47]:
```suggestion
		STAT_COG = 1
```

File: modular_sojourn/general/oddity.dm [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+/obj/item/oddity/roundstart/book
+	name = "story book"
+	desc = "A familiar book depicting a strange and alien exo-planet, telling its stories by images and strange-looking words. Maybe your ancestry came from here... Or maybe you found this world.... many possibilities."
+	icon_state = "rounstart_oddity_1"
+
+	oddity_stats = list(
+		STAT_MEC = 1,
+		STAT_BIO = 1,
+		STAT_COG = 1,
```
Comment (Trilbyspaceclone) [inline modular_sojourn/general/oddity.dm:36]:
```suggestion
		STAT_COG = 1
```

File: modular_sojourn/general/oddity.dm [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+	)
+
+/obj/item/oddity/roundstart/book_2
+	name = "old book"
+	desc = "A book depicting a specific type of story , by using narratives and images. Theres alot of general informations that teach the reader about old and ancient traditions and tales."
```
Comment (Trilbyspaceclone) [inline modular_sojourn/general/oddity.dm:41]:
```suggestion
	desc = "A book depicting a specific type of story, by using narratives and images. Theres alot of general informations that teach the reader about old and ancient traditions and tales."
```

**Comment by Guidesu** [comment]
> Not really sure why they'd be oddities, this just seems like something that psions will use to powergamer for round start weapons/tools.

I mean.
Its oddities becuase its adds some value of not being a useless trinket.
And even if they do that. They just used a whole point on the loadout to get an RNG based item or gun or tool that could be used somewhere else.
If they do that its probably more about their thing than anything because the guy just destroyed a trinket that was or given by their family or found or used for a really long time.

**Comment by r4d6** [comment]
> > Not really sure why they'd be oddities, this just seems like something that psions will use to powergamer for round start weapons/tools.
> 
> I mean. Its oddities becuase its adds some value of not being a useless trinket. And even if they do that. They just used a whole point on the loadout to get an RNG based item or gun or tool that could be used somewhere else. If they do that its probably more about their thing than anything because the guy just destroyed a trinket that was or given by their family or found or used for a really long time.

They can randomize the stats and get a higher result than the current 1-1-1, with the folken elders straight up increasing all stats, guaranteed.

**Comment by Guidesu** [comment]
> > > Not really sure why they'd be oddities, this just seems like something that psions will use to powergamer for round start weapons/tools.
> > 
> > 
> > I mean. Its oddities becuase its adds some value of not being a useless trinket. And even if they do that. They just used a whole point on the loadout to get an RNG based item or gun or tool that could be used somewhere else. If they do that its probably more about their thing than anything because the guy just destroyed a trinket that was or given by their family or found or used for a really long time.
> 
> They can randomize the stats and get a higher result than the current 1-1-1, with the folken elders straight up increasing all stats, guaranteed.

Do you have any suggestion then?

**Comment by r4d6** [comment]
> Do you have any suggestion then?

Well if they're supposed to be fluff, then not making them oddities would work.

Also, 1 loadout points out of 10 isn't a lot if the person doesn't have 8 different loadout items. So it would be easy for a psion to just see it as a free tool or weapon, if a random one, especially since otherwise we don't have any loadout-based tool that you can take, baring duct tape.

**Comment by Guidesu** [comment]
> > Do you have any suggestion then?
> 
> Well if they're supposed to be fluff, then not making them oddities would work.
> 
> Also, 1 loadout points out of 10 isn't a lot if the person doesn't have 8 different loadout items. So it would be easy for a psion to just see it as a free tool or weapon, if a random one, especially since otherwise we don't have any loadout-based tool that you can take, baring duct tape.

Theres anyway to code that it makes psions not able to transform oddity into item?

**Comment by cdb-is-not-good** [comment]
The answer is almost always yes, but why? If it's just for fluff there's zero good reason to have it be an oddity.

**Comment by Guidesu** [comment]
W

> The answer is almost always yes, but why? If it's just for fluff there's zero good reason to have it be an oddity.

There is.
Its just more fun because its an trinket instead of a random ass item that does nothing.

**Comment by cdb-is-not-good** [comment]
That is by definition not 'fluff'.

**Pull request closed without merging**